<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CharacterSelectionClass</class>
 <widget class="QWidget" name="CharacterSelectionClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CharacterSelection</string>
  </property>
  <widget class="QPushButton" name="button_selectCharacter">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>200</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Character</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_back">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>260</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QListView" name="list_character">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>321</width>
     <height>511</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="button_createNewCharacter">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>200</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Create Character</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_characterName">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>170</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>140</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>New Character Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Character</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
